apps/backend/models/forumPost.ts

```diff
@@ -0,0 +1,86 @@
+import { list } from '@keystone-6/core';
+import { text, timestamp, relationship } from '@keystone-6/core/fields';
+
+export const ForumPost = list({
+  access: {
+    operation: {
+      query: ({ session }) => !!session,
+      create: ({ session }) => !!session,
+      update: ({ session }) => !!session,
+      delete: ({ session }) => !!session,
+    },
+    filter: {
+      // only logged-in user can see posts
+      query: ({ session }) => {
+        if (!session?.data?.id) return false;
+        return true;
```

Comment:
There isn't anything inherently wrong with this logic but we do have some access control redundancy here

For now my only suggestion is to refactor to a simplified way to write this code, as shown below:

```suggestion
      query: ({ session }) => !!session?.data?.id
```